JMeterAKSPipeline/CreateLoadTestInfra.ps1

```diff
@@ -0,0 +1,47 @@
+param(
```

Comment:
Shouldn't we move all scripts in Scripts folder ??

Reply:
These are scripts for consumption from pipeline and best to have it in Pipeline folder.
All manual scripts (for consumption of users from local machines) are in Scripts folder.